test/e2e/scheduling/preemption.go

```diff
@@ -241,7 +247,7 @@ var _ = SIGDescribe("SchedulerPreemption [Serial]", func() {
 			framework.Logf("Created pod: %v", pods[i].Name)
 		}
 		if len(pods) < 2 {
-			e2eskipper.Skipf("We need at least two pods to be created but" +
+			framework.Failf("We need at least two pods to be created but" +
```

Comment:
I'm fine with this change, it should have been this way to begin with.  This probably landed before the verify job started enforcing the no-skips requirement for conformance

Reply:
Yeah, that is a nice point.
All conformance tests should be operated without considering any conditions.
So it seems fine to replace Skipf with Failf to know the conformance test is operated always.

BTW I found a small thing here.
We need to put a space after `but to distinguish it from later `all` word.

Reply:
/lgtm

anyways :-)